### ISSUE TITLE
Issues opening project

### ISSUE THREAD

**Issue by chrisdrogaris** [Unity-Technologies/arfoundation-demos#44]
I am having issues opening in Unity in 2020.1.0.a11 and also tried a newer version.

I get the following error:
Library/PackageCache/com.unity.device-simulator@2.2.3-preview/Editor/Shims/ApplicationSimulation.cs(6,44): error CS0246: The type or namespace name 'ApplicationShimBase' could not be found (are you missing a using directive or an assembly reference?)

Any ideas?

**Comment by DanMillerDev** [comment]
This is an issue with the device simulator package and compatibility. You can navigate to package manager and remove the device simulator package to fix this.

I use the device simulator package to test the onboarding UX for different supported devices.

I'll look into different version or removing it entirely, thanks for flagging!

**Comment by Nouraaaaaan** [comment]
I have an issue with Google Pixel 4XL phone, the AR camera is not opened and remains black.

**Comment by chrisdrogaris** [comment]
I removed device sumulator and get this error:
Library/PackageCache/com.unity.localization@0.7.1-preview/Editor/Reporting/ProgressReporter.cs(22,16): error CS0246: The type or namespace name 'Progress' could not be found (are you missing a using directive or an assembly reference?)

**Comment by DanMillerDev** [comment]
@chrisdrogaris sorry for the delay. I've updated and removed some packages. The latest master branch uses AR Foundation 4.1.5. Since Unity 2020.2 has been released we currently do not support Unity 2020.1. These are considered Tech streams and we only support the latest tech stream.

There are some issues with XR Plugin management and it requires Unity 2019.4.15+

If you pull the latest version and upgrade your project to 2020.2 if should compile and work fine. There's also an [additional note](https://github.com/Unity-Technologies/arfoundation-demos#building-for-unity-20202) in the readme about configuring for Android and ARCore.
